### PR TITLE
Detects whether a participant is jigasi and sets a property.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: JitsiConference.js [hunk 1]
```diff
@@ -1351,12 +1351,26 @@ JitsiConference.prototype.onMemberJoined = function(
         JitsiConferenceEvents.USER_JOINED,
         id,
         participant);
+
+    const updateFeatures = features => {
```
Comment (hristoterezov) [inline JitsiConference.js]: Can't we avoid having nested function?

File: JitsiConference.js [hunk 2]
```diff
@@ -1351,12 +1351,26 @@ JitsiConference.prototype.onMemberJoined = function(
...
+    };
+
     this.xmpp.caps.getFeatures(jid)
-        .then(features => {
-            participant._supportsDTMF = features.has('urn:xmpp:jingle:dtmf:0');
-            this.updateDTMFSupport();
-        },
-        error => logger.warn(`Failed to discover features of ${jid}`, error));
+        .then(updateFeatures)
+        .catch(_ => { // eslint-disable-line no-unused-vars
```
Comment (hristoterezov) [inline JitsiConference.js]: Why the unused var? maybe () => {...} will help fixing the eslint errors, isn't it?

File: JitsiConference.js [hunk 3]
```diff
@@ -1351,12 +1351,26 @@ JitsiConference.prototype.onMemberJoined = function(
...
+            // there was probably a mismatch, lets try one more time and givup
```
Comment (hristoterezov) [inline JitsiConference.js]: Why do we do the retry?

File: JitsiConference.js (hunk 2 above)
Comment (damencho) [reply JitsiConference.js]: Oh, yes.

File: JitsiConference.js (hunk 3 above)
Comment (damencho) [reply JitsiConference.js]: Cause we always hit the validation problem that Lenny recently fixed: https://github.com/jitsi/lib-jitsi-meet/commit/a89d67388f49cc18aae080cb388877380a7664b1

Comment (hristoterezov) [reply JitsiConference.js]: But how we are hitting it since it is fixed?

**Pull request merged**